.github/checksum.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+RESULT_FILE=$1
+
+if [ -f $RESULT_FILE ]; then
+  rm $RESULT_FILE
+fi
+touch $RESULT_FILE
+
+checksum_file() {
+  echo $(openssl md5 $1 | awk '{print $2}')
```

Comment:
I'm not even sure whether it's relevant, but can we change md5 to sha256 here for peace of mind?

Reply:
It doesn't matter but SHA-256 won't hurt either. Pushed a fix.